server/client.go

```diff
@@ -320,12 +341,16 @@ func (c *client) sendInfo(info []byte) {
 
 func (c *client) sendErr(err string) {
 	c.mu.Lock()
-	if c.bw != nil {
+	if c.bw != nil && !c.internal {
```

Comment:
Internal Clients should be send only, meaning the send to others, so they should not have a bw (buffered writer). So is this not needed?